ui/partials/main/thGlobalTopNavPanel.html

```diff
@@ -10,6 +10,7 @@
       </button>
       <ul class="dropdown-menu" role="menu" aria-labelledby="th-logo">
         <li><a href="perf.html" class="dropdown-item">Perfherder</a></li>
+        <li><a href="intermittentsview.html" class="dropdown-item">Intermittents View</a></li>
```

Comment:
@edmorley and @camd: What about "Intermittent Failures"? Perhaps a bit more self-explanatory?

Reply:
I'm not great at naming things, so I've posted to the mailing list/CCed a few people :-)
https://groups.google.com/forum/#!topic/mozilla.tools.treeherder/_m4sscUO2Po

(Would be good to get this figured out now, since changes later mean breaking URLs, confusing tools that don't follow Git renames, confusing Bugzilla since not everything can be renamed when renaming a component etc)